modules.d/90mdraid/mdraid_start.sh

```diff
@@ -65,6 +70,14 @@ _md_force_run() {
             _md_start "${_md}"
         done
     fi
+
+    # try to assemble all the left md devices that should be assembled 
```

Comment:
Remove white space after the word assembled:

https://github.com/dracut-ng/dracut-ng/actions/runs/9638464635/job/26579189718?pr=410